ampd/src/event_sub/stream.rs

```diff
@@ -95,6 +95,8 @@ where
         })
         .collect::<Result<Vec<_>>>()?;
 
+    tokio::time::sleep(Duration::new(1, 0)).await;
```

Comment:
```suggestion
    tokio::time::sleep(Duration::from_secs(1)).await;
```

Reply:
This also isn't the best place to add the delay. This function is wrapped in a `with_retry` and makes no sense to force every failure take >1 secs. It should be added in the `update` method of `BlockState` after the while loop.